search_frontend/js/search.js

```diff
@@ -30,20 +31,50 @@
           'c':''
       };
 
-      this._nodeSearchData = ko.observableArray([]);
+      this._nodeSearchData = ko.observableArray();
+      this.structureData = ko.observableArray()
       this._page = ko.observable(0);
       this._total = ko.observable(0);
       this._activateTab = ko.observable(false);
     }
-
-    /** @todo fetch from api */
+        /* @todo find what type of node is the current tab*/
+    findStructure() {
+      var list = [] 
+        for (var key in mockup_data.filterData) {
+          if (key == this._node_type ) { 
+            for (var x in mockup_data.filterData[key]) 
+              list.push(x)   
+            }
+              this.structureData.push(list)  
+          }
+    } 
+
+    fillInformation(row) {
+      var list = [] 
+      var start = 0 
+      var end = this.structureData()[0].length - 1 
+       for( var key in row) {
+          if ( start <= end) {
+            var row1 = row[this.structureData()[0][start]]
+            list.push(row1)
+            start++;
+          }
+        }
+          this._nodeSearchData.push(list);
+          this.structureData.push(list)
+          start= 0;
+          list = []
+      }
+        /** @todo fetch from api */
     fetch () {
-      let data = mockup_data[this._node_type].data;
-      data.forEach(row => {
-        this._nodeSearchData.push(row);
-      });
-      this._page(this._page()+1);
-    }
+          this.findStructure(mockup_data.filterData); 

```

Comment:
Each time we call fetch, findStructure is executed. This should be done in the constructor.